### PR TITLE
Refactor createDashboard method in e2e-tests

### PR DESCRIPTION
**Pull request by mournfulCoroner**

### CODE REVIEW

File: tests/opensource-suites/dash/selectors/datetimeSelectors.test.ts [hunk 1]
```diff
@@ -1,47 +1,46 @@
...
-        );
-        await dashboardPage.waitForSelector(
-            `${slct(ControlQA.chartkitControl)} input[value='${expectedValues[1]}']`,
-        );
-
-        await deleteEntity(page, WorkbooksUrls.E2EWorkbook);
-    });
+    datalensTest(
+        'Date/time selectors should display the time',
+        async ({page}: {page: Page; config: TestParametrizationConfig}) => {
```
Comment (mournfulCoroner) [inline tests/opensource-suites/dash/selectors/datetimeSelectors.test.ts:17]: Need to remove `config: TestParametrizationConfig`

File: src/ui/units/dash/components/DashActionPanel/EditControls/EditControls.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import {Button, Icon} from '@gravity-ui/uikit';
 import block from 'bem-cn-lite';
 import {I18n} from 'i18n';
 import {Feature} from 'shared';
+import {ActionPanelDashSaveControls} from 'shared/constants/qa/action-panel';
```
Comment (goshander) [inline src/ui/units/dash/components/DashActionPanel/EditControls/EditControls.tsx:8]: Missed QA suffix

File: tests/opensource-suites/dash/base/relations.test.ts [hunk 3]
```diff
@@ -85,7 +73,7 @@ datalensTest.describe('Dashboards - Basic functionality', () => {
                 return elems.length === 1;
             });
 
-            await deleteEntity(page, WorkbooksUrls.E2EWorkbook);
+            await dashboardPage.deleteDash();
```
Comment (goshander) [inline tests/opensource-suites/dash/base/relations.test.ts:76]: Maybe you can use `afterEach` callback for deleting entity. Its looks more safety

**Review by goshander — DISMISSED** [review]

File: tests/opensource-suites/dash/base/actionPanel.test.ts [hunk 4]
```diff
@@ -1,27 +1,31 @@
...
+import {WorkbookIds} from '../../../constants/constants';
 
 const PARAMS = {
     CONTROL_TITLE: 'test-control',
     CONTROL_FIELD_NAME: 'test-control-field',
 };
 
 datalensTest.describe('Dashboards - Basic functionality', () => {
+    datalensTest.beforeEach(async ({page}: {page: Page}) => {
+        // some page need to be loaded so we can get data of feature flag from DL var
```
Comment (Marginy605) [inline tests/opensource-suites/dash/base/actionPanel.test.ts:17]: You need to write this comment in every case before openTestPage(page, '/');

**Review by Marginy605 — APPROVED** [review]

File: tests/opensource-suites/dash/base/relations.test.ts (hunk 3 above)
Comment (mournfulCoroner) [reply tests/opensource-suites/dash/base/relations.test.ts:77]: The flag isEnabledHideOldRelations allows you to complete the test without creating a dash, which prevents you from using this hook

**Review by goshander — APPROVED** [review]

**Pull request merged**